graphdatascience/tests/unit/conftest.py

```diff
@@ -20,7 +20,7 @@
 from graphdatascience.session.dbms_connection_info import DbmsConnectionInfo
 
 # Should mirror the latest GDS server version under development.
-DEFAULT_SERVER_VERSION = ServerVersion(2, 6, 0)
+DEFAULT_SERVER_VERSION = ServerVersion(2, 10, 0)
```

Comment:
>under development

We are developing 2.12.0 of the plugin now. 2.11.0 is the latest stable.
Should this be 2.12 or 2.11?

Reply:
now with monthly updates, this can get quite annoying to update.
As its only a version for unit-testing we could also just use a very high number